src/pages/BusinessPage/components/BusinessDetails/index.jsx

```diff
@@ -25,13 +25,15 @@ export default function BusinessDetails({ businessData, userPosition }) {
                 </div>
 
                 <div className={style['business-rating']}>
+                    <span className={style['reviewrs-count']}>Based on {businessData.reviews.length}  reviews</span>
```

Comment:
so this is hard coded text, if you saw all over the app we use `<FormattedMessage />` component, which comes from `react-intl` that translates our app.

[here is a link for the documention of <FormattedMessage />](https://github.com/formatjs/react-intl/blob/master/docs/Components.md#formattedmessage)

at the end it should look something like this
```html
<FormattedMessage
  id="Reviews amount"
  values={{
    reviews: businessData.reviews.length,
  }}
/>
```